openedx/core/djangoapps/content/course_views/tabs.py

```diff
@@ -0,0 +1,573 @@
+"""
+Tabs for courseware.
+"""
+from abc import abstractmethod
+
+from xmodule.tabs import CourseTab, key_checker, need_name
+from openedx.core.lib.plugins.api import CourseViewType
+from courseware.access import has_access
+from student.models import CourseEnrollment
+from ccx.overrides import get_current_ccx
+
+_ = lambda text: text
+
+
+def is_user_staff(course, user):
```

Comment:
These helper methods are unrelated to tabs, so I'd move them into a util file. It's also possible that they aren't really needed anymore. I introduced them to hide the ugly import logic, but it might be clearer to just call the code directly now.